functions/knative/wasmedge/http-server/README.md

```diff
@@ -0,0 +1,100 @@
+# Sample Function WasmEdge ( written in Rust )
+
+## Prerequisites
+
+### OpenFunction
+
+You can refer to the [Installation Guide](https://github.com/OpenFunction/OpenFunction#install-openfunction) to setup OpenFunction.
+
+## Run it locally
+
+1. Build
+
+```bash
+cargo build --target wasm32-wasi --release
+```
+
+2. Run
+
+```bash
+wasmedge target/wasm32-wasi/release/wasmedge_hyper_server.wasm
+```
+
+3. Test
+
+Run the following from another terminal.
+
+```bash
+$ curl http://localhost:8080/echo -X POST -d "WasmEdge"
+WasmEdge
+```
+
+## Deployment
+
+1. Create secret
```

Comment:
@wrongerror you can follow https://openfunction.dev/docs/concepts/serverless_apps/ to revise the guide, no need to add redundant contents to this and we need a wasmedge guide in openfunction.dev as well